### ISSUE TITLE
"make install" fails

### ISSUE THREAD

**Issue by whitemarvin** [phillipberndt/fakexrandr#42]
"make install" fails, and I get the following errors:

> TARGET_DIR=`sed -nre 's/#define FAKEXRANDR_INSTALL_DIR "([^"]+)"/\1/p' config.h`; \
> [ -d $TARGET_DIR ] || exit 1; \
> install libXrandr.so $TARGET_DIR; \
> ln -s libXrandr.so $TARGET_DIR/libXrandr.so.2 || true; \
> ln -s libXrandr.so $TARGET_DIR/libXinerama.so.1 || true; \
> ldconfig
> install: cannot create regular file '/usr/local/lib/libXrandr.so': Permission denied
> ln: failed to create symbolic link '/usr/local/lib/libXrandr.so.2': Permission denied
> ln: failed to create symbolic link '/usr/local/lib/libXinerama.so.1': Permission denied
> /sbin/ldconfig.real: Can't create temporary cache file /etc/ld.so.cache~: Permission denied
> Makefile:19: recipe for target 'install' failed
> make: *** [install] Error 1

Can I install without doing sudo? I'm on Ubuntu 16.04. Thanks.

**Comment by phillipberndt** [comment]
No, you can't. You can however copy the generated libXrandr.so to any directory, create the symlinks manually, and add the directory to your library search path by adding `export LD_LIBRARY_PATH=/full/path` to your user's bashrc.

**Comment by whitemarvin** [comment]
I see. It would be a time-saver to have an option for this in the "make install". An idea for a feature maybe?

Anyway, thanks!

**Comment by nmschulte** [comment]
I haven't used fakexrandr in a while, but I've got this change stashed from nearly 2 years ago.

```
nmschulte@desmas-l:~/src/fakexrandr$ g show stash
```
```
commit b51d3aa51002936d59e28414b12fff0eb7f14d9a (refs/stash)
Merge: 07c42c6 5eff876
Author: Nathan Schulte <nmschulte@gmail.com>
Date:   Thu Feb 4 10:03:08 2016 -0600

    On xcb: local build modifications

diff --cc Makefile
index bf83c19,bf83c19..2321194
--- a/Makefile
+++ b/Makefile
@@@ -1,4 -1,4 +1,4 @@@
--PREFIX=/usr
++PREFIX=/home/nmschulte/.local
  CFLAGS=-O2
  
  ifeq ($(shell pkg-config --errors-to-stdout --print-errors xcb-randr),)
diff --cc configure
index 24467e8,24467e8..565130d
--- a/configure
+++ b/configure
@@@ -28,7 -28,7 +28,7 @@@ REAL_LIBRARY_DIR=$(dirname "${REAL_LIBR
  
  # 3) Determine location for fake xrandr library
  LIBRARY_DIRECTORIES=($(ldconfig -v 2>/dev/null | grep -oE '^/[^:]+'))
--FAKE_LIBRARY_DIRECTORY=
++FAKE_LIBRARY_DIRECTORY=/home/nmschulte/.local/lib
  CANDIDATES=
  for DIR in ${LIBRARY_DIRECTORIES[@]}; do
        if [ "$DIR" == "${REAL_LIBRARY_DIR}" ]; then
```
